src/library/scala/deprecated.scala

```diff
@@ -55,7 +55,21 @@ import scala.annotation.meta._
  *  @see    [[scala.deprecatedInheritance]]
  *  @see    [[scala.deprecatedOverriding]]
  *  @see    [[scala.deprecatedName]]
+ *  @see    [[scala.deprecatedError]]
  */
 @getter @setter @beanGetter @beanSetter @field
-@deprecatedInheritance("Scheduled for being final in 2.14", "2.13.0")
+@deprecatedInheritance("Scheduled for being sealed in 2.14", "2.13.0")
 class deprecated(message: String = "", since: String = "") extends scala.annotation.StaticAnnotation
+
+/** An annotation that designates that an annottee should no longer be used.
```

Comment:
We must correct the spelling of the neologism. `annotatee` (what is annotated by an annotator) or `annotee` (anglicization).

Other good words from checking the OED: annonary (provisions), annotine (year-old), annosity (fullness of years), annoyously.

Reply:
Sample contemporary usage of annonary: https://sandus.org/central-peoples-government/office-of-the-sogmo/nobility-honours/fraternal-annonary-order-of-the-wooden-bowl/

Reply:
The Annotated. An undercover grad student and a mole in Lightbend attempt to identify each other while combing through scala-internals threads for the originalist interpretation of Scala Language Spec.